### PR TITLE
Posture counts

### PR DESCRIPTION
**Pull request by gnmahanth**
Fixes #

Changes proposed in this pull request:
- `scan_count` should be number of accounts which are scanned irrespective of active or inactive accounts 
- trigger `CachePostureProviders` task on detecting compliance scan success, this fixes the delay in sync of cached posture counts
-

### CODE REVIEW

**Review by ramanan-ravi — APPROVED** [review]

File: deepfence_worker/cronjobs/cloud_compliance.go [hunk 1]
```diff
@@ -237,9 +238,9 @@ func CachePostureProviders(ctx context.Context, task *asynq.Task) error {
 
 			scan_count_query = `
 			MATCH (n:` + string(neo4jNodeType) + `)
-			WHERE n.pseudo=false and n.active=true and n.agent_running=true
```
Comment (jatin-baweja) [inline deepfence_worker/cronjobs/cloud_compliance.go:240]: Is the condition `n.agent_running=true` not required for linux posture count too?

File: deepfence_worker/ingesters/common.go [hunk 2]
```diff
@@ -156,3 +167,17 @@ func ToMap[T any](data T) map[string]interface{} {
 	_ = json.Unmarshal(out, &bb)
 	return bb
 }
+
+func anyCompleted(data []map[string]interface{}) bool {
+
+	complete := false
+
+	for i := range data {
+		if data[i]["scan_status"].(string) == utils.SCAN_STATUS_SUCCESS {
```
Comment (jatin-baweja) [inline deepfence_worker/ingesters/common.go:176]: Do we need to check if string conversion gave no error? Otherwise, might run into runtime error

Comment (gnmahanth) [reply deepfence_worker/ingesters/common.go:176]: we dont need to check here as this field is guaranteed to have string data

File: deepfence_worker/cronjobs/cloud_compliance.go (hunk 1 above)
Comment (gnmahanth) [reply deepfence_worker/cronjobs/cloud_compliance.go:240]: hosts with agent_running=false are not allowed to scan so they will not appear here

**Pull request merged**